src/app/page.tsx

```diff
@@ -3,10 +3,20 @@
 import { HeaderPage } from '@/Features/header';
 import { PromoCard } from '@/components/PromoCard';
 
+import { Footer } from '@/components/Footer';
+
 const Home = async () => {
-  const response = await fetch('http://localhost:3000/api/barbershops');
-  const jsonResponse = await response.json();
-  const numberOfBarbershops = jsonResponse.data;
+  const barbershopsInfoResponse = await fetch('http://localhost:3000/api/barbershopsInfo');
+  const barbershopsInfoJsonResponse = await barbershopsInfoResponse.json();
```

Comment:
let's do it more convinient: `const {branchesTotal, countriesTotal, capitalsTotal} = await barbershopsInfoResponse.json();`

numberOfSomething - I find this naming too long and not informative, let's stick to original version